### ISSUE TITLE
Error please fix it

### ISSUE THREAD

**Issue by Magicmk330** [AbdeslemSmahi/NetflixChecker#17]
It's showing -

ProgressBar.__init__() got an unexpected keyword argument 'max_value'
An error occurred.. Saving progress...

**Comment by Lunar808** [comment]
pip3 install prograssbar2

**Comment by massimodeg** [comment]
@Can you explain to me how to use it?

**Comment by Lunar808** [comment]
> @can you explain to me how to use it?

execute it on terminal, or cmd if you're using Windows(taking to account you installed python) 
pip is the package installer that comes with python, progressbar2 is the package that you're missing and that's the reason you've got this error.

It's pretty straight forward. your question tells me you probably don't know a lot of python(sorry if I'm mistaken, don't mean to be rude.) so I recommend you to learn a bit, it will  help you fix this problems with ease and even contribute to projects in github like this project.

**Comment by massimodeg** [comment]
yes, you guessed it, i don't know python, if you could explain how to use it, you would help me a lot...i just started studying python

**Comment by Lunar808** [comment]
> yes, you guessed it, i don't know python, if you could explain how to use it, you would help me a lot...i just started studying python

I pretty much explained.

>>pip3 install prograssbar2
run in terminal(or cmd if using windows)
pip3 = python package manager
install - sub-command to install package(install "enter package")
progressbar2 - the package itself that you're missing, and that is the reason you have this error.

good luck with learning python, it's a great scripting language and very useful overall.
P.S search google about pip, it will help you tremendously in the future. it's mandatory for python developing.
